packages/gatsby/src/query/index.js

```diff
@@ -163,7 +163,7 @@ const processQueries = async (
   { activity, graphqlRunner, graphqlTracing }
 ) => {
   const queue = queryQueue.createBuildQueue(graphqlRunner, { graphqlTracing })
-  await queryQueue.processBatch(queue, queryJobs, activity)
+  return queryQueue.processBatch(queue, queryJobs, activity)
```

Comment:
Any reason why we don't keep this void?

Reply:
We're using the return in the state machine

Reply:
what does it return? I'm pretty sure it returns undefined.

Reply:
Returns the results, we added those in the queue handlers

Reply:
This returns undefined in master so that's why

Reply:
Oh right, that change was omitted from this PR I think